frontend/javascripts/oxalis/view/right-border-tabs/starting_job_modals.tsx

```diff
@@ -446,17 +463,24 @@ export function MaterializeVolumeAnnotationModal({
   const tracing = useSelector((state: OxalisState) => state.tracing);
   const activeSegmentationTracingLayer = useSelector(getActiveSegmentationTracingLayer);
   const fixedSelectedLayer = selectedVolumeLayer || activeSegmentationTracingLayer;
+  const readableVolumeLayerName =
+    fixedSelectedLayer && getReadableNameOfVolumeLayer(fixedSelectedLayer, tracing);
+  const hasFallbackLayer =
+    fixedSelectedLayer && "tracingId" in fixedSelectedLayer
+      ? fixedSelectedLayer.fallbackLayer != null
+      : false;
   const isMergerModeEnabled = useSelector(
     (state: OxalisState) => state.temporaryConfiguration.isMergerModeEnabled,
   );
   let description = (
     <p>
       Start a job that takes the current state of this volume annotation and materializes it into a
-      new dataset. All annotations done on the
-      {` "${fixedSelectedLayer && getReadableNameOfVolumeLayer(fixedSelectedLayer, tracing)}" `}
-      volume layer will be merged with the data of the fallback layer.
+      new dataset.
+      {hasFallbackLayer
+        ? ` All annotations done on the "${readableVolumeLayerName}" volume layer will be merged with the data of the fallback layer. `
+        : null}
       {isMergerModeEnabled
-        ? " Additionally, the skeletons will be used to merge segments as merger mode is active. "
+        ? " Since merger mode is currently active, in the new output dataset the segments connected via skeleton nodes will be merged. "
```

Comment:
```suggestion
        ? " Since the merger mode is currently active, the segments connected via skeleton nodes will be merged within the new output dataset."
```